rgw/admin/utils.go

```diff
@@ -0,0 +1,61 @@
+package admin
+
+import (
+	"fmt"
+	"net/url"
+	"reflect"
+	"strings"
+)
+
+const (
+	queryAdminPath = "/admin"
+)
+
+func buildQueryPath(endpoint, path, args string) string {
+	return fmt.Sprintf("%s%s%s?%s", endpoint, queryAdminPath, path, args)
+}
+
+// getReflect encodes structs into URL query parameters.
```

Comment:
`getValues` I suppose? Also, it may be even better to give it a name closer to what it does? Something like: `valueToURLParams` ?